src/graphql/query/shared/hasEverLoggedIn.graphql

```diff
@@ -1,3 +1,13 @@
 query hasEverLoggedIn {
 	hasEverLoggedIn @client
+	my {
+		userAccount {
+			id
+			balance
+			isFirstTimeDepositor
+		}
+		loans {
+			totalCount
+		}
+	}
```

Comment:
Yeah, I'm concerned about that too. These were requested as general controls that can be used in optimizely, not for a specific experiment, but we could ask if we could limit this to a certain page initially. 

I think we could also try skipping the query if the cookies already exist, that way it only happens for the first request in the session.

Reply:
If we can focus this in or prevent subsequent queries I think that would be helpful. Otherwise, we'll just have to keep an eye out for any lengthening of the page load...